app/mailers/journeys/creation/journey_mailer.rb

```diff
@@ -0,0 +1,37 @@
+module Journeys
+  module Creation
+    class JourneyMailer < ApplicationMailer
+
+      def confirmation_of_booking(recipient, journey)
+        @user = journey.user
+        @journey = journey
+        @car = @journey.car
+        
+        from = select_from
```

Comment:
* rename `select_from` to `from`
* don't create local variable, just use a single method call: `mail(from: from, to: recipient, subject: 'Booking confirmation')`